components/automate-load-balancer/habitat/config/nginx.conf

```diff
@@ -231,7 +231,6 @@ http {
       proxy_pass https://automate-gateway;
       # Required to make persistent connections happen
       proxy_set_header Connection "";
-      add_header Strict-Transport-Security "max-age=63072000; includeSubDomains" always;
```

Comment:
Why have we removed these headers?

Reply:
Because now we have added this header directly in the services. And load balancer was again adding same header. So we were getting duplicate header at client side (UI).